config/locales/views/genders/en.yml

```diff
@@ -0,0 +1,52 @@
+en:
+  genders:
+    male: 'Male'
+    Female: 'Female'
+    Transgender: 'Transgender'
+    --: ' -- '
```

Comment:
Uh, what's this @rachaelkiteme?  Is that coming from the database?  @trystant what is this supposed to represent, and how does it differ from other?

Reply:
@rlgreen91 yes got that from the database.
![screenshot from 2018-03-31 23-01-09](https://user-images.githubusercontent.com/16720356/38211626-bf74d8b4-36c3-11e8-87b9-7e60cff1456c.png)